### ISSUE TITLE
Spawning Children

### ISSUE THREAD

**Issue by RedSparr0w** [Adrianotiger/desktopPet#10]
How do you always spawn child in front of your Pet?

For example:
Below image (the bottom child) i have set position x to `imageX-80` which spawns infront if pet is walking to the left, otherwise it will spawn behind, have also tried `imageX+80` which has the same result except to the opposite direction.
![image](https://cloud.githubusercontent.com/assets/7288322/23631829/8c6fc8da-0324-11e7-8603-bda5a9d6e30f.png)
Edit:
Solved this part used `imageX+imageW` somehow works.

**Comment by RedSparr0w** [comment]
### Spawning multiple children at once?
I am trying to spawn a 6 part image
![](https://i.imgur.com/4rrVhlE.png)

But there does not seem to be an easy/nice looking way to do this.

what im currently doing:
```xml
<child animationid="20">
  <x>imageX+imageW*1.5</x>
  <y>imageY</y>
  <next>22</next>
</child>
<child animationid="22">
  <x>imageX</x>
  <y>imageY-80</y>
  <next>24</next>
</child>
<child animationid="24">
  <x>imageX</x>
  <y>imageY-80</y>
  <next>26</next>
</child>
<child animationid="21">
  <x>imageX+imageW*2.5</x>
  <y>imageY</y>
  <next>23</next>
</child>
<child animationid="23">
  <x>imageX</x>
  <y>imageY-80</y>
  <next>25</next>
</child>
<child animationid="25">
  <x>imageX</x>
  <y>imageY-80</y>
  <next>27</next>
</child>
```
but this makes each child appear a bit after eachother.

What would work better: _(Gives an error)_
```xml
<child animationid="20">
  <x>imageX+imageW*1.5</x>
  <y>imageY</y>
  <next>22</next>
</child>
<child animationid="20">
  <x>imageX+imageW*1.5</x>
  <y>imageY-80</y>
  <next>24</next>
</child>
<child animationid="20">
  <x>imageX+imageW*1.5</x>
  <y>imageY-160</y>
  <next>26</next>
</child>
<child animationid="20">
  <x>imageX+imageW*2.5</x>
  <y>imageY</y>
  <next>23</next>
</child>
<child animationid="20">
  <x>imageX+imageW*2.5</x>
  <y>imageY-80</y>
  <next>25</next>
</child>
<child animationid="20">
  <x>imageX+imageW*2.5</x>
  <y>imageY-160</y>
  <next>27</next>
</child>
```

**Comment by Adrianotiger** [comment]
You are the second one this week, reporting this bug 😄 
Of course you are right, it should be possible to add more than 1 child at the same time. I will try to solve this bug this week and I will release it as soon as possible.

**Comment by Adrianotiger** [comment]
I found the problem and I will release an update in the next 24 hours. Can you publish it as beta? So I can check the new version and see if it works.

**Comment by Adrianotiger** [comment]
Solved on version 1.0.5

**Comment by RedSparr0w** [comment]
Damn that was quick, Thanks!
Works perfectly!
